esignet-injiweb.properties

```diff
@@ -413,18 +413,18 @@ mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.vci.identif
   'credential_definition': {\
   'type': {'VerifiableCredential'},\
   'credentialSubject': {\
-  'fullName': { 'display': {{'name': 'Full Name', 'locale': 'en' }}},\
-  'phone': { 'display': {{'name': 'Phone Number', 'locale': 'en' }}},\
-  'dateOfBirth': { 'display': {{'name': 'DOB', 'locale': 'en' }}},\
-  'gender': { 'display': {{'name': 'Gender', 'locale': 'en' }}},\
-  'residenceStatus': { 'display': {{'name': 'Residence Status', 'locale': 'en' }}},\
-  'email': { 'display': {{'name': 'Email Id', 'locale': 'en' }}},\
-  'region': { 'display': {{'name': 'Region', 'locale': 'en' }}},\
-  'province': { 'display': {{'name': 'Province', 'locale': 'en' }}},\
-  'city': { 'display': {{'name': 'City', 'locale': 'en' }}},\
-  'postalCode': { 'display': {{'name': 'Postal Code', 'locale': 'en' }}}\
+  'fullName': {'display': {{'name': 'Name','locale': 'en'}}}, \
+  'mobile': {'display': {{'name': 'Phone Number','locale': 'en'}}},\
+  'dob': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
+  'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
+  'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
+  'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
+  'policyIssuedOn': {'display': {{'name': 'Policy Issued On','locale': 'en'}}},\
+  'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
+  'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
+  'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
    }},\
-  'display': {{'name': 'MOSIP Identity Verifiable Credential', \
+  'display': {{'name': 'MOSIP Verifiable Credential', \
```

Comment:
this should be Health Insurance Verifiable Credential, and the other one should be Life Insurance Verifiable Credential. Lets not use MOSIP in the name, since it is not releated to the identity.

Reply:
changed this